src/model_theory/basic.lean

```diff
@@ -59,19 +59,36 @@ structure language :=
 (functions : ℕ → Type u) (relations : ℕ → Type v)
 
 /-- Used to define `first_order.language₂`. -/
-def sequence₂ (a₀ a₁ a₂ : Type u) : ℕ → Type u
+@[simp] def sequence₂ (a₀ a₁ a₂ : Type u) : ℕ → Type u
 | 0 := a₀
 | 1 := a₁
 | 2 := a₂
 | _ := pempty
 
 instance {a₀ a₁ a₂ : Type u} [h : inhabited a₀] : inhabited (sequence₂ a₀ a₁ a₂ 0) := h
 
+instance {a₀ a₁ a₂ : Type u} {n : ℕ} : is_empty (sequence₂ a₀ a₁ a₂ (n + 3)) := pempty.is_empty
```

Comment:
I think it would be nice to add the instances for `inhabited (sequence₂ a₀ a₁ a₂ 1)` and `inhabited (sequence₂ a₀ a₁ a₂ 2)` too.